.travis.yml

```diff
@@ -5,9 +5,23 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
```

Comment:
Maybe something we can add in another PR since it would probably introduce `allow_failures` also.